pyproject.toml

```diff
@@ -46,7 +46,7 @@ seaborn = "^0.12"
 scikit-learn = "^1"
 watchdog = "^3"
 tqdm = "^4"
-pandas = "^1"
```

Comment:
No pandas?

Reply:
it's uh, extinction or something idk Darwin's theory.